v0/util/index.js

```diff
@@ -155,6 +156,17 @@ const formatTimeStamp = (dateStr, format) => {
   }
 };
 
+// Check what type of gender and convert to f or m
+
+const hashToSha256ForGender = gender => {
+  if (gender.toUpperCase() === "FEMALE" || gender.toUpperCase() === "F") {
```

Comment:
Should we consider `woman` and `man` as well (similar to Braze)?

Reply:
done